src/core.jl

```diff
@@ -209,7 +221,26 @@ Ez(cosmo::Cosmology, z) = _Ez(cosmo.cosmo, z)
 Hmpc(cosmo::Cosmology, z) = cosmo.cosmo.h*Ez(cosmo, z)/CLIGHT_HMPC
 comoving_radial_distance(cosmo::Cosmology, z) = cosmo.chi(z)
 growth_factor(cosmo::Cosmology, z) = cosmo.Dz(z)
-function power_spectrum(cosmo::Cosmology, k, z)
-    Dz2 = growth_factor(cosmo, z)^2
-    @. exp(cosmo.lplk(log(k)))*Dz2
+omega_x(cosmo::Cosmology, z, species_x_label) = _omega_x(cosmo.cosmo, z, species_x_label)
```

Comment:
Where did the `_omega_x` function go? I can't see it anywhere

Reply:
https://github.com/damonge/LimberJack.jl/blob/c1fbc8d1ed06734eb2182d1f86b7d7ab01033188/src/halofit.jl#L119-L123

Reply:
ok, then:
1. If `_omega_x` does not exist, this line should disappear. I think there's a similar one in `LimberJack.jl`.
2. The implementation in halofit.jl is wrong (confirmed with Andrina). It should read
```
    omegaDEwz = (1 - cpar.Ωm)/Ez^2
```

Reply:
Ok great! implemented!